internal/cli/status.go

```diff
@@ -1179,6 +1179,8 @@ func (c *StatusCommand) getWorkspaceFromProject(pr *pb.GetProjectResponse) (stri
 	var workspace string
 
 	if len(pr.Workspaces) != 0 {
+		// TODO (clint): verify this is fine with the removal of default
```

Comment:
Did you do this? 😄

Reply:
I have now! Thank you for the reminder. I had to make some adjustments in dbf8ed63 please take a look when you can :D